py/server/deephaven_internal/stream.py

```diff
@@ -0,0 +1,89 @@
+import io
+
+
+class TeeStream(io.TextIOBase):
+    """
+    TextIOBase subclass that splits output between a delegate instance and a set of lambdas. Can delegate some calls to
+    the provided stream but actually write only to the lambdas.
+
+    Useful to adapt any existing file-like object (such as sys.stdout or sys.stderr), but actually let some other
+    processing take place before writing.
+    """
+
+    encoding = 'UTF-8'
+    closed = False
```

Comment:
These are variables attached to the *class* not the object instance. I'm assuming that's not what we want?

Should be set in constructor.

Reply:
Because these are declared (in C, as read-only), these must be re-declared at class level, or else they cannot be assigned in the constructor (or other methods).

There might be a better way to work around a C superclass, but this is what @jmao-denver directed.

Reply:
I'm confused, b/c `closed` should be a property of each `TeeStream`, not a global value. I'll take a look at TextIOBase later.

Reply:
Looks like this isn't needed after all - my best guess is that this was an artifact of the other bugs we were running into with jpy.